.travis.yml

```diff
@@ -40,6 +40,10 @@ addons:
       - gcc-multilib
       - lib32gcc-6-dev
 
+before_script:
+  - wget http://releases.linaro.org/components/toolchain/binaries/6.3-2017.02/aarch64-linux-gnu/gcc-linaro-6.3.1-2017.02-x86_64_aarch64-linux-gnu.tar.xz
```

Comment:
@rleon I intentionally used an explicit reference for the toolchain version in order to avoid potential issue introduced by updates from Linaro side. Also they don't have version file under the latest, so I would have to run curl + awk/sed to extract the name/version.

But overall, I don't have a strong opinion about this. If maintainers prefer to go with latest, I can update the script.